.idea/runConfigurations/Artemis__Server__LocalVC___LocalCI__IRIS_.xml

```diff
@@ -2,7 +2,7 @@
   <configuration default="false" name="Artemis (Server, LocalVC &amp; LocalCI, IRIS)" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
     <option name="ACTIVE_PROFILES" value="dev,localci,localvc,artemis,scheduling,local,iris" />
     <option name="ALTERNATIVE_JRE_PATH" value="17" />
-    <module name="de.tum.in.www1.artemis.Artemis.main" />
+    <module name="Artemis.main" />
```

Comment:
What's up with that, what's the correct value here? It seems like there's a commit war putting in either of those

Reply:
Must have pushed that by accident, will remove it

Reply:
Mh I mean, I always have to set `Artemis.main`, so I'm fine with that, just want to find out why this is happening and for some it's the first, and for some the second value which is working